models/tags.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var Db = require('/../config/database');
+
+var TagSchema = mongoose.Schema({
+    title          : {type: String, required: true},
+    coordinates: {
+        lat        : {type: Number, required: true},
+        lon        : {type: Number, required: true}
+    },
+    tag            : {type: String, required: true},
+    owner          : {type: String, required: true},
+    creationDate   : {type: Date  , default : Date.now},
+    expirationDate : {type: Date  , required: true},
+    score          : {type: Number, default : 0}
+});
```

Comment:
We decided today to add a location field to the Tag Mongo Model, so can you add the follow:
`location: {type: String, required: true}`